Sprint #13 Homework

ekalemen opened the pull request:
This adds Player layout and inflate it with data from SearchActivity.

Commit:
Sprint #13 Homework

This adds Player layout and inflate it with data from SearchActivity.

Signed-off-by: Eugeny Kalemenev <e.kalemenev@yadro.com>

DmitrijEgorow commented:
- [ ] Цвет боковых кнопок не соответствует дизайн-макету
<img width="651" alt="Screenshot 2025-01-15 at 11 29 19" src="https://github.com/user-attachments/assets/ea471528-3230-45d8-8025-7b778de3a362" />

DmitrijEgorow commented:
- [ ] Разметка "поехала"
![Screenshot_20250115_112351](https://github.com/user-attachments/assets/28bc6450-c78c-4959-b5e5-215804d43065)

DmitrijEgorow commented on app/src/main/res/layout/activity_player.xml:
```diff
@@ -0,0 +1,280 @@
+        app:layout_constraintGuide_end="24dp" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/leftGuide16"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_begin="16dp" />
+
+    <androidx.constraintlayout.widget.Guideline
+        android:id="@+id/rightGuide16"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:layout_constraintGuide_end="16dp" />
+
+    <ImageView
+        style="@style/SettingsTopHeaderImageView"
+        android:id="@+id/player_button_back"
```
Разные стили именования идентификаторов: где-то через camelCase, где-то через _

DmitrijEgorow commented on app/src/main/res/layout/activity_player.xml:
```diff
@@ -0,0 +1,280 @@
+        android:text="@string/track_album"
+        android:textColor="@color/search_gray"
+        android:textFontWeight="400"
+        android:textSize="13sp"
+        app:layout_constraintStart_toEndOf="@id/leftGuide16"
+        app:layout_constraintTop_toBottomOf="@id/durationLabel"
+        app:layout_constraintBottom_toTopOf="@id/yearLabel" />
+
+    <TextView
+        android:id="@+id/yearLabel"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:fontFamily="@font/ys_display_regular"
+        android:letterSpacing="0"
+        android:text="@string/track_year"
+        android:textColor="@color/search_gray"
+        android:textFontWeight="400"
+        android:textSize="13sp"
```
Стоит вынести res-values

ekalemen commented:
> * [ ]  Цвет боковых кнопок не соответствует дизайн-макету

Да, у меня почему то не получилось на темной теме использовать drawable src для ImageView с транспарентным каналом (rgba), фон кнопок просто черный тоже. Поробую разобраться.

UPD: Разобрался, исправил

ekalemen commented:
> * [ ]  Разметка "поехала"

Странно, на эмуляторе(Pixel6 API35) выглядит нормально. Из-за чего такое может быть? Разная высота экранов?

UPD: я понял в чем дело и поправил верстку

![image](https://github.com/user-attachments/assets/e9d7c42d-e755-4b91-8085-c1dc90c89c5a)

Commit:
Sprint #13 homework review fixes

This add fixes for review issues of sprint #13 homework:
 - fix player layout for small screens
 - fix colors of the player buttons
 - show placeholder for tracks without album cover

Signed-off-by: Eugeny Kalemenev <e.kalemenev@yadro.com>
